app/src/main/java/com/odysee/app/adapter/NotificationListAdapter.java

```diff
@@ -202,8 +202,7 @@ public void onBindViewHolder(NotificationListAdapter.ViewHolder vh, int position
         vh.titleView.setVisibility(!Helper.isNullOrEmpty(notification.getTitle()) ? View.VISIBLE : View.GONE);
         vh.titleView.setText(notification.getTitle());
         vh.bodyView.setText(notification.getDescription());
-        vh.timeView.setText(DateUtils.getRelativeTimeSpanString(
-                getLocalNotificationTime(notification), System.currentTimeMillis(), 0, DateUtils.FORMAT_ABBREV_RELATIVE));
+        vh.timeView.setText(FormatTime.fromEpochMillis(getLocalNotificationTime(notification)));
```

Comment:
I'm not sure if `getLocalNotificationTime` is still needed, and if it might make the time shown wrong.

Reply:
If I understand correctly, `fromEpochMillis` checks from January 1st, 1970 UTC time, right? In that case, you can remove the call to  `getLocalNotificationTime` as this converts the notification time (in UNIX time, UTC timezone) to the local time based on the device timezone.